app/abci.go

```diff
@@ -6,15 +6,15 @@ import (
 	"fmt"
 	"sort"
 
+	"github.com/celestiaorg/celestia-app/x/celestiaapp/types"
```

Comment:
After reading your 1st comment and going though the PR, I'm wondering if this is feasable to rename the path to 
`github.com/celestiaorg/celestia-app/x/celestia-app/types` to follow with what we have for celestia-appd
Example of what we are doing it here `github.com/celestiaorg/celestia-app/cmd/celestia-appd/cmd`

Reply:
I like that option, but was worried about having nested directories with the same name. Also, celestia-appd is not meant to be imported, but celestaiaapp will have to be imported, so the extra - would automatically force packages that import it to rename that import. We should only ever need to include the PayForMessage stuff in it, so maybe we could just name the module `payment`?

we could also go with John's suggestion and call it `celestiapp`

wdyt?

Reply:
I like John's suggestion for the rest. Still for this case, payment is more detailed considering your awesome explanation. `payment` lgtm 🌟

Reply:
:+1:  resolved in https://github.com/celestiaorg/lazyledger-app/pull/118/commits/b36a13dd83a71196219dd12718a537f6ee1bfeed